cmd/kube-proxy/app/server.go

```diff
@@ -421,17 +628,17 @@ func (s *ProxyServer) Run() error {
 	return nil
 }
 
-func getConntrackMax(config *options.ProxyServerConfig) (int, error) {
-	if config.ConntrackMax > 0 {
-		if config.ConntrackMaxPerCore > 0 {
-			return -1, fmt.Errorf("invalid config: ConntrackMax and ConntrackMaxPerCore are mutually exclusive")
+func getConntrackMax(config componentconfig.KubeProxyConntrackConfiguration) (int, error) {
+	if config.Max > 0 {
+		if config.MaxPerCore > 0 {
+			return -1, fmt.Errorf("invalid config: Conntrack Max and Conntrack MaxPerCore are mutually exclusive")
 		}
 		glog.V(3).Infof("getConntrackMax: using absolute conntrack-max (deprecated)")
-		return int(config.ConntrackMax), nil
+		return int(config.Max), nil
 	}
-	if config.ConntrackMaxPerCore > 0 {
-		floor := int(config.ConntrackMin)
-		scaled := int(config.ConntrackMaxPerCore) * runtime.NumCPU()
+	if config.MaxPerCore > 0 {
```

Comment:
The function signature has changed to `getConntrackMax(config componentconfig.KubeProxyConntrackConfiguration)`, where `config` is as `KubeProxyConntrackConfiguration`.